Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "nitro-cli"
-version = "1.0.8"
+version = "1.0.9"
```

Comment:
Should add also the update in Cargo.lock (https://github.com/aws/aws-nitro-enclaves-cli/blob/main/Cargo.lock#L923).

Reply:
Right, will update it in the next revision.